worlds/shorthike/Options.py

```diff
@@ -61,6 +61,21 @@ class CostMultiplier(Range):
     range_end = 200
     default = 100
 
+class FillerCoinAmount(Choice):
+    """The amount of coins that will be in each filler coin item."""
```

Comment:
This should be number and not amount
```suggestion
    """The number of coins that will be in each filler coin item."""
```

Reply:
Oh, I've been not criticizing this on other PRs because I see "amount" used this way _constantly_, literally several times a day, and at that point it just feels like at best a poor use of everyone's time to comment on every single one of them. Maybe we need a separate PR to fix all the existing ones?